sphinx/ext/intersphinx/_load.py

```diff
@@ -107,15 +107,19 @@ def fetch_inventory_group(
     app: Sphinx,
     now: int,
 ) -> bool:
-    cache_time = now - app.config.intersphinx_cache_limit * 86400
+    if app.config.intersphinx_cache_limit != -1:
+        cache_time = now - app.config.intersphinx_cache_limit * 86400
+    else:
+        cache_time = None
     failures = []
     try:
         for inv in invs:
             if not inv:
                 inv = posixpath.join(uri, INVENTORY_FILENAME)
             # decide whether the inventory must be read: always read local
             # files; remote ones only if the cache time is expired
-            if '://' not in inv or uri not in cache or cache[uri][1] < cache_time:
+            if '://' not in inv or uri not in cache or \
+                    (cache_time and cache[uri][1] < cache_time):
```

Comment:
```suggestion
            if (
            	'://' not in inv or uri not in cache or
                (cache_time and cache[uri][1] < cache_time)
            ):
```

Using line continuations makes it harder to read (and I think we mostly use '()' now).